netlify.toml

```diff
@@ -0,0 +1,25 @@
+# Sentry uses netlify to deploy pull-request previews of the frontend which
+# talk to the production API.
+#
+# This is an EXPERIMENTAL developer experience, there are still various pages
+# that are served through django and have no SPA equivalent views.
+
+[build]
+  publish = "src/sentry/static/sentry/dist"
+  command = "yarn webpack"
+
+[[redirects]]
+  from = "/_assets/*"
```

Comment:
I have `from = "/sentry/dist/*"` instead of `from = "/_assets/*"` on my branch. That said don't know which one is needed (if any at all).

Reply:
`_assets` will be right, since I changed where webpack serves files from